tools/render/templates/policy.adoc

```diff
@@ -13,7 +14,7 @@
 Category:: {{ index $annotations "policies.kyverno.io/category" }}
 Minimum Kyverno version:: {{ index $annotations "policies.kyverno.io/minversion" }}
 Subject:: {{ index $annotations "policies.kyverno.io/subject" }}
-Policy type:: "{{ .Type }}"
+Policy types:: {{ range $i, $e := .Types }}`{{ . }}`{{ if ne (plus1 $i) $typecount }}, {{ end }}{{ end }}
```

Comment:
I would probably done that a bit differently, but I can't argue with the result :see_no_evil: 

My attempt at this would be 
```
Policy types:: {{ range $i, $e := .Types }}{{ if $i }}, {{ end }}`{{ $e }}`{{ end }}
```
But this is surprisingly hard to do..

Reply:
Thanks for the hint, I was wondering if there's really no way to do this without having a custom template function.